### PR TITLE
Remove remaining enzyme tests

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Description
Remove remaining enzyme tests

## Testing done
Unit testing and code coverage

## Screenshots
N/A

## Acceptance criteria
- [ ] All test should pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/components/VAOSApp/index.unit.spec.jsx [hunk 1]
```diff
@@ -156,4 +156,30 @@ describe('VAOS <VAOSApp>', () => {
...
+    });
+
+    await waitFor(() => {
+      expect(
+        screen.getByText(
+          /We’re sorry. We can’t find any VA medical facility registrations for you/,
+        ),
+      ).to.be.ok;
+    });
+    screen.debug();
```
Comment (jbalboni) [inline src/applications/vaos/tests/components/VAOSApp/index.unit.spec.jsx:183]: Missed removing this, looks like

**Review by jbalboni — APPROVED** [review]

File: src/applications/vaos/tests/components/VAOSApp/index.unit.spec.jsx (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/tests/components/VAOSApp/index.unit.spec.jsx]: Thought I got them all!!!

**Comment by jbalboni** [comment]
@vbahinwillit It looks like FormButtons from the list in the ticket wasn't removed

**Comment by jbalboni** [comment]
@vbahinwillit Also, while it wasn't in the list in the ticket, did we cover FacilityDirectionsLink in the FacilityAddress ticket? If so, we could remove that one as well.

**Review by timwright12 — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -18525,9 +18525,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#8a9ef5109d833dcc65a1bf2c2ae785c3cccb1466":
-  version "18.1.0"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#8a9ef5109d833dcc65a1bf2c2ae785c3cccb1466"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#c12156e77d9cbffa439ee0ec2ddf42b805658023":
+  version "18.4.0"
```
Comment (jbalboni) [inline yarn.lock:18529]: We definitely don't want to check in this change, it bumps the schema version which other forms depend on

**Review by jbalboni — CHANGES_REQUESTED** [review]

**Review by jbalboni — APPROVED** [review]

**Pull request merged**